### PR TITLE
issue-2604-changing-completed-project-labels-via-if-statement

### LINKED ISSUE #2604: Need to change "Current Project Team" with "Project Team"
### Overview
When projects are completed, there is no longer a "current project team". Therefore, we need to change the words from "Current Project Team" to "Project Team" on completed projects.

### Action Items
<details>
<summary>Picture Summary of Changes needed</summary>
If the status is completed, the h1 tag should say "Project Team" instead of "Current Project Team"
Screen Shot 2022-01-07 at 1 12 44 PM
</details>

- [x] Find the h1 tag that is attributed to the "Current Project Team" header in the [project template](https://github.com/hackforla/website/blob/gh-pages/_layouts/project.html)
     - [x] Use liquid if statement to check if the status of the project is completed
     - [x] Make sure you're using this methodology: if project status is completed, h1 tag should say "Project Team", else h1 tag should say "Current Project Team"

### Resources/Instructions
*Note: Best way to see all of the completed projects is to search through the completed filtered projects on the project page of HFLA. 

[Completed Projects Page](https://www.hackforla.org/projects/?status=Completed)

Issue Creator - [Tomas Gonzalez - Slack DM](https://hackforla.slack.com/app_redirect?channel=U02DF1PPQUA)

### PR DESCRIPTION
**Pull request by Georgema20**
Fixes #2604 

### What changes did you make and why did you make them ?

 I found the h1 tag that is attributed to the "Current Project Team" header in the project template, used liquid if statement to check if the status of the project is completed, and made sure to use this methodology: if the project status is completed, h1 tag should say "Project Team", else h1 tag should say "Current Project Team". 

I did this because completed projects should have "Project Team" instead of "Current Project Team" as their title. 

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

screencapture-hackforla-org-projects-hellogov-2022-01-20-19_13_09
screencapture-hackforla-org-projects-ems-triage-tracker-2022-01-20-19_11_14
screencapture-hackforla-org-projects-heart-2022-01-20-19_12_51
screencapture-hackforla-org-projects-undebate-2022-01-20-19_14_17
screencapture-hackforla-org-projects-metro-ontime-2022-01-20-19_14_00
screencapture-hackforla-org-projects-work-for-la-2022-01-20-19_14_37
screencapture-hackforla-org-projects-public-tree-map-2022-01-20-19_13_24

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
screencapture-localhost-4000-projects-work-for-la-2022-01-20-19_28_18
screencapture-localhost-4000-projects-public-tree-map-2022-01-20-19_27_18
screencapture-localhost-4000-projects-hellogov-2022-01-20-19_27_00
screencapture-localhost-4000-projects-ems-triage-tracker-2022-01-20-19_26_11
screencapture-localhost-4000-projects-undebate-2022-01-20-19_28_03
screencapture-localhost-4000-projects-metro-ontime-2022-01-20-19_27_43
screencapture-localhost-4000-projects-heart-2022-01-20-19_26_38

</details>

### CODE REVIEW

**Comment by JessicaLucindaCheng** [comment]
Removed `Ready for Milestone` label as the issue associated with this PR was already in the Prioritized Backlog (before it was moved to the In Progress column) and had a milestone already.

**Comment by JessicaLucindaCheng** [comment]
- Availability: 6 hours
- Review ETA: Tuesday, January 25, 2022

File: _layouts/project.html [hunk 1]
```diff
@@ -105,7 +105,11 @@ <h2 class='project-page-card-title title6'>Resources</h2>
 
     <!-- Project Team Card -->
     <div id='project-team-card' class='page-card card-primary page-card-lg page-card--project-description-page'>
-        <h2 class='project-page-card-title title6'>Current Project Team</h2>
+        {% if page.status == "Completed" %}
+             <h2 class='project-page-card-title title6'> Project Team</h2>
```
Comment (JessicaLucindaCheng) [inline _layouts/project.html]: 1. Right before `<h2`, please remove 1 space.
2. For`> Project Team`, there is an extra space between `>` and `Project Team`. Please remove that space.

File: _layouts/project.html [hunk 2]
```diff
@@ -105,7 +105,11 @@ <h2 class='project-page-card-title title6'>Resources</h2>
...
+        {% else %}
+              <h2 class='project-page-card-title title6'>Current Project Team</h2>
```
Comment (JessicaLucindaCheng) [inline _layouts/project.html]: Right before `<h2`, please remove 2 spaces.

**Review by JessicaLucindaCheng — CHANGES_REQUESTED** [review]
@Georgema20 Thanks for working on this issue! Please make the changes as suggested in my inline comments. Also, please do the following:

1. Please add your pull request to the project board.
Add-to-project-board

2. Please edit the title of your pr so it has a descriptive title of what you have done. For more info, see this: https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#i-complete-pull-request-1-update-pull-request-title

File: _layouts/project.html [hunk 3]
```diff
@@ -105,7 +105,11 @@ <h2 class='project-page-card-title title6'>Resources</h2>
...
+        {%  endif %}
```
Comment (JessicaLucindaCheng) [inline _layouts/project.html]: Currently, there is an extra space in `{%  endif %}`. Please remove 1 space between `{%` and `endif %}`

**Comment by anthonysim** [comment]
ETA: 1/29/22
Availability: 3 hrs, Saturday 12-3 PM

**Comment by anthonysim** [comment]
@Georgema20, LGTM! Please make the changes that @JessicaLucindaCheng suggested and its good to go!

**Comment by Georgema20** [comment]
Availability: 1 hour
ETA: midnight 1/26

**Comment by Georgema20** [comment]
@JessicaLucindaCheng hey I think I did it all. Let me know if you want to see anything different.

**Comment by JessicaLucindaCheng** [comment]
- Availability: 6 hours
- Review ETA: Thu, Jan 27,2022

**Comment by JessicaLucindaCheng** [comment]
@Georgema20 Sorry, I don't have time to review it today. So, I will be delaying my review by one day.
- Availability: 6 hours
- ETA: Friday, January 28, 2022

**Review by JessicaLucindaCheng — APPROVED** [review]
@Georgema20 Your changes look good to me!

**Review by anthonysim — APPROVED** [review]

**Pull request merged**